### PR TITLE
Adds Old insuls and Old gas mask among assistant heirlooms

### PR DESCRIPTION
**Pull request by eeSPee**
## About The Pull Request

Adds ~~worn down insuls~~ and old gas mask among assistant heirlooms.

## Why It's Good For The Game

Gotta nail that classic assistant look!

## Changelog
:cl:
add: Old gas mask among assistant heirlooms
/:cl:

### CODE REVIEW

**Comment by ToasterBan** [comment]
aren't those roundstart insuls?

**Comment by Sinestia** [comment]
> aren't those roundstart insuls?

They're basically budget insulated gloves but a lot better

3/7 chance of just being insuls
3/7 chance to be half protection
and 1/7 chance to be 25% protection.

**Comment by ivanmixo** [comment]
Potentially having roundstart insuls is bad. Family heirloom is already problematic as a trait, this just makes it worse.

**Comment by Sinestia** [comment]
Also https://github.com/BeeStation/BeeStation-Hornet/pull/4255


Old gas mask isn't bad

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -134,7 +134,7 @@
 			if("Chaplain")
 				heirloom_type = pick(/obj/item/toy/windupToolbox, /obj/item/reagent_containers/food/drinks/bottle/holywater)
 			if("Assistant")
-				heirloom_type = /obj/item/storage/toolbox/mechanical/old/heirloom
+				heirloom_type = pick(/obj/item/storage/toolbox/mechanical/old/heirloom,/obj/item/clothing/mask/gas/old,/obj/item/clothing/gloves/color/fyellow/old)
```
Comment (Archanial) [inline code/datums/traits/negative.dm]:
```suggestion
				heirloom_type = pick(/obj/item/storage/toolbox/mechanical/old/heirloom, /obj/item/clothing/mask/gas/old, /obj/item/clothing/gloves/color/fyellow/old)
```

**Comment by bloons3** [comment]
So you see, _fingerless_ insulated gloves are legitimately useless and only cosmetic.

The gloves you want to add, are not. Please do not give assistants roundstart insulated gloves.

**Comment by ike709** [comment]
No roundstart insuls.

**Comment by eeSPee** [comment]
> So you see, _fingerless_ insulated gloves are legitimately useless and only cosmetic.
> 
> The gloves you want to add, are not. Please do not give assistants roundstart insulated gloves.

Ok got that.

**Review by qwertyquerty — CHANGES_REQUESTED** [review]
No roundstart insuls

**Comment by eeSPee** [comment]
Aite done.

**Comment by bloons3** [comment]
Bit concerned that giving assistants gas masks roundstart will encourage them to look like greytiders all round long.

Heirloom items shouldn't be _useful_, since carrying them around is meant to be annoying to balance the point they give you.

**Comment by ike709** [comment]
No roundstart gasmask.

**Comment by Sinestia** [comment]
> Bit concerned that giving assistants gas masks roundstart will encourage them to look like greytiders all round long.
> 
> Heirloom items shouldn't be _useful_, since carrying them around is meant to be annoying to balance the point they give you.

You aren't necessarily wrong but there is quite a disparity

**Comment by eeSPee** [comment]
Not a good idea.

**Pull request closed without merging**